src/executables/master_heartbeat.c

```diff
@@ -328,6 +330,13 @@ static HB_JOB_FUNC hb_resource_jobs[] = {
         "  Error Logging: disabled\n"
 #define HA_ADMIN_INFO_NOLOG_EVENT_FORMAT_STRING  \
         "    %s\n"
+
+#define HA_FAILBACK_DIAG_STRING       "[Failback] [Diagnosis]"
+#define HA_FAILBACK_SUCCESS_STRING    "[Failback] [Success]"
+#define HA_FAILBACK_CANCEL_STRING     "[Failback] [Cancelled]"
+#define HA_FAILOVER_DIAG_STRING       "[Failover] [Diagnosis]"
+#define HA_FAILOVER_SUCCESS_STRING    "[Failover] [Success]"
+#define HA_FAILOVER_CANCEL_STRING     "[Failover] [Cancelled]"
```

Comment:
prefix와 message 사이의 공백을 여기에 추가하는게 어떨까요?

```suggestion
#define HA_FAILBACK_DIAG_STRING       "[Failback] [Diagnosis] "
#define HA_FAILBACK_SUCCESS_STRING    "[Failback] [Success] "
#define HA_FAILBACK_CANCEL_STRING     "[Failback] [Cancelled] "
#define HA_FAILOVER_DIAG_STRING       "[Failover] [Diagnosis] "
#define HA_FAILOVER_SUCCESS_STRING    "[Failover] [Success] "
#define HA_FAILOVER_CANCEL_STRING     "[Failover] [Cancelled] "
```

Reply:
수정하였습니다.